docs/config.md

```diff
@@ -140,3 +138,12 @@ chains.tekton.dev/transparency-upload: "true"
 | `signers.kms.auth.oidc.role`      | Role used for OIDC authentication                                                           |                  |
 | `signers.kms.auth.spire.sock`     | URI of the Spire socket used for KMS token (e.g. `unix:///tmp/spire-agent/public/api.sock`) |                  |
 | `signers.kms.auth.spire.audience` | Audience for requesting a SVID from Spire                                                   |                  |
+
+## Supporting Namespace Restrictions in Chains Controller
```

Comment:
Consider re-writing this section to only describe the feature once, then explain how to use it, and provide an example:

> ## Namespace Restrictions
> 
> By default, Chains will process TaskRun and PipelineRun resources in all namespaces. This may not be ideal for certain use cases. To restrict the namespaces processed by Chains use the `--namespace` parameter in the Chains controller. The value for this parameter is a comma separated list of namespaces, e.g. `--namespace default,my-app`.